lib/ember_cli/build_monitor.rb

```diff
@@ -36,7 +36,11 @@ def complete?
     end
 
     def build_error?
-      error_file.exist? && error_file.size?
+      error_file.exist? && error_file.size? && not(has_deprecation_warnings?)
```

Comment:
If the error file contains a deprecation warning (and it always will in the case of the deprecation warning from #464) then this `not(has_deprecation_warnings?)` check will mean that even if the error file contains actual build errors as well `build_error?` will still respond `false`.

Instead the check should verify whether the error file contains any lines that are not a deprecation warning.